sources/platform/actors/development/programming_interface/container_web_server.md

```diff
@@ -14,6 +14,10 @@ import TabItem from '@theme/TabItem';
 
 Each Actor run is assigned a unique URL (e.g. `kmdo7wpzlshygi.runs.apify.net`) that allows HTTP access to an optional web server running inside the Actor's Docker container. This feature enhances your Actor's capabilities by enabling external communication.
 
+:::tip
+The container web server provides a way how to connect to one specific Actor run. To enable using your Actor as an API, with a pre-defined hostname, load balancing and autoscaling, check out [Actor Standby](./actor_standby.md).
+:::
```

Comment:
```suggestion
:::tip Container connectivity

The container web server provides a way how to connect to one specific Actor run. To enable using your Actor as an API, with a pre-defined hostname, load balancing and autoscaling, check out [Actor Standby](./actor_standby.md).

:::
```

just a small suggestion for styling the admonitions & the title

Reply:
Thanks! Will do.